library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.ts

```diff
@@ -60,7 +60,7 @@ export class BtnProfileComponent extends WidgetBaseComponent
     super()
     this.btnAppsConfig = { ...this.basicBtnAppsConfig }
     this.btnSettingsConfig = { ... this.settingBtnConfig }
-    if (this.configSvc.unMappedUser.profileDetails.profileStatus === 'VERIFIED') {
+    if (this.configSvc.unMappedUser && this.configSvc.unMappedUser.profileDetails.profileStatus === 'VERIFIED') {
```

Comment:
please put additional condition for "this.configSvc.unMappedUser.profileDetails"